engine/apps/telegram/views.py

```diff
@@ -1,11 +1,8 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
-from apps.telegram.tasks import register_telegram_webhook
```

Comment:
Moving this to [`on_worker_ready` ](https://github.com/grafana/oncall/pull/1984/files#diff-4f09c06cda07f53f96a39ae416e18ce7abc654598ef625f21b05b92758cdce1bR56-R62) so it doesn't fail with RabbitMQ connectivity error when building the docker image.